### PR TITLE
Departure v4.2

### PR DESCRIPTION
**Pull request by sauloperez**
This prepares the full rename to Departure for Rails v4.2.

### CODE REVIEW

File: departure_error.log [hunk 1]
```diff
@@ -0,0 +1 @@
+No foreign keys reference `percona_migrator_test`.`comments`; ignoring --alter-foreign-keys-method.
```
Comment (enricostano) [inline departure_error.log:1]: Is this intented to be committed?

Comment (sauloperez) [reply departure_error.log:1]: ups no, it should be ignored like in other branches. Thanks for the headsup!

**Comment by benlangfeld** [comment]
Is there something preventing this from being merged?

**Comment by sauloperez** [comment]
Hi @benlangfeld ! actually not much. I'll move it forward and release it ASAP.

**Comment by sauloperez** [comment]
I just published the gem https://rubygems.org/gems/departure/versions/4.0.0

**Comment by benlangfeld** [comment]
Thanks @sauloperez!

**Pull request merged**